packages/fuel-indexer-types/src/lib.rs

```diff
@@ -29,6 +29,8 @@ pub mod prelude {
     pub use crate::{TypeId, FUEL_TYPES_NAMESPACE};
 }
 
+pub use tai64::Tai64;
```

Comment:
Oh whoops, I knew something felt wrong. 👍🏽

Reply:
Since type aliases can't be used as constructors, I opted to do:

```rust
quote! {
    .unwrap_or(Tai64Timestamp::from({
        <[u8;8]>::try_from(0u64.to_be_bytes()).expect("Failed to create byte slice from u64")
    }))
}
```

as it's not possible to do:

```rust
quote! {
    .unwrap_or(Tai64Timestamp(0u64))
}
```